debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerStreamingChangeEventSource.java

```diff
@@ -61,14 +61,14 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(LogMinerStreamingChangeEventSource.class);
 
+    // TODO: change this to use MAX_QUEUE_SIE as the default
```

Comment:
Log an issue for tracking?

Reply:
I'm actually going to make this change as a part of this PR.  In fact, it should be sufficient enough to remove the constant entirely and replace it with `connectorConfig.getMaxQueueSize()`.